tests/unit/core/server/request/test-class-core-get-reporting-payment-activity-request.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Class Get_Reporting_Payment_Activity_Test
+ *
+ * @package WooCommerce\Payments\Tests
+ */
+
+use PHPUnit\Framework\MockObject\MockObject;
+use WCPay\Core\Exceptions\Server\Request\Invalid_Request_Parameter_Exception;
+use WCPay\Core\Server\Request\Get_Reporting_Payment_Activity;
+
+/**
+ * WCPay\Core\Server\Request unit tests.
+ */
+class Get_Reporting_Payment_Activity_Test extends WCPAY_UnitTestCase {
+
+	/**
+	 * Mock WC_Payments_API_Client.
+	 *
+	 * @var WC_Payments_API_Client|MockObject
+	 */
+	private $mock_api_client;
+	/**
+	 * Mock WC_Payments_API_Client.
```

Comment:
⛏️ code comment correction
```suggestion
	 * Mock WC_Payments_Http_Interface.
```